CHANGELOG.md

```diff
@@ -25,6 +25,7 @@
 - PR #454 Improve CSV reader docs and examples
 - PR #465 Added templated C++ API for RMM to avoid explicit cast to `void**`
 - PR #472 RMM: Created centralized rmm::device_vector alias and rmm::exec_policy
+- PR #617 Introduced .dockerignore file to project. Prevents stale cmake cache files from being "ADD" to the docker container if previously built on host machine which causes the container build to fail.
```

Comment:
Shorter message is better here -- can always refer to GitHub PR for full details.